bayesian_exercises/election_poll/election_poll.jl

```diff
@@ -0,0 +1,176 @@
+### A Pluto.jl notebook ###
+# v0.11.12
+
+using Markdown
+using InteractiveUtils
+
+# ╔═╡ 036183d2-f775-11ea-0976-eb09b0c00f66
+begin
+	using Soss
+	using Distributions
+	using Plots
+end
+
+# ╔═╡ 5e231d96-f778-11ea-3c30-f134d4b4b537
+md"## Using Soss.jl to analize an election poll"
+
+# ╔═╡ 87ccbd50-f778-11ea-3451-97f4184db99d
+md"We have data about the preferences of 1,447 likely voters:
+
+Bush: 727
+Dukakis: 583
+Other: 137
+"
+
+# ╔═╡ f1fcfc00-f778-11ea-322a-95b2444823ff
+begin
+	data = [727,583,137]
+	n = sum(data)
+	k = length(data)
+	p = data ./n
+end;
+
+# ╔═╡ 02721cc8-f781-11ea-0894-67ee2caeea30
+n
+
+# ╔═╡ 25630eb6-f779-11ea-0779-1f36f318865f
+md"Modelating this data as a Multinomial Random Variable, with each probability of success as a Dirichlet distribution.
+
+The dirichlet distribution can be seen as a generalization of the beta distribution. Initializing it as a non-informative uniform, we have:" 
```

Comment:
You can add reference links with markdown, i.e. `[Dirichlet Distribution](https://en.wikipedia.org/wiki/Dirichlet_distribution)`